### ISSUE TITLE
[Bug] The landscape mode is not working properly

### ISSUE THREAD

**Issue by tolotrasamuel** [chess-analysis-heat-map/feedback#3]
Sometimes, in landscape mode, the piece score are too small

**Comment by 0rkaM** [comment]
I agree since there is plenty of unused space the board should be mechanics to the full possible extent, together with pieces and texts. Also reducing, or better removing the the top title in Analysis mode seems the obvious thing to do.
